crates/oxc_linter/src/rules/eslint/no_invalid_regexp.rs

```diff
@@ -54,12 +55,13 @@ declare_oxc_lint!(
     NoInvalidRegexp,
     eslint,
     correctness,
+    config = NoInvalidRegexpConfig,
 );
 
-#[derive(Debug, Clone, Deserialize, Default)]
+#[derive(Debug, Clone, Deserialize, Default, JsonSchema)]
+#[serde(rename_all = "camelCase", default)]
 struct NoInvalidRegexpConfig {
-    #[serde(default, rename = "allowConstructorFlags")]
```

Comment:
I'm assuming this change works fine based on the tests and everything, but I'm not 100% sure so please make sure this looks fine when reviewing.